integrations/tensorboard/Neptune-TensorBoard.ipynb

```diff
@@ -0,0 +1,765 @@
+    ]
+   },
+   "source": [
+    "# Convert TensorBoard logs to Neptune experiments"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "7MizxtDs5dQX",
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Before you start"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "-ydwJoVD6Je9",
+    "tags": [
+     "installation"
+    ]
+   },
+   "source": [
+    "### Install Dependencies"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "id": "BFnMHkiXfU7-",
+    "tags": [
+     "installation"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install --quiet tensorboard==2.4.0 tensorflow==2.3.1 neptune-tensorboard==0.5.1 neptune-client==0.4.129"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "installation",
+     "library_updates"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "! pip install --quiet --upgrade tensorboard neptune-tensorboard neptune-client"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "### Create some TensorBoard logs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "comment"
+    ]
+   },
+   "source": [
+    "**Note:**  \n",
+    "You can run it once to create a single TensorBoard log or multiple times."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "tags": [
+     "code"
+    ]
+   },
+   "outputs": [],
+   "source": [
+    "import tensorflow as tf\n",
+    "import datetime\n",
+    "\n",
+    "mnist = tf.keras.datasets.mnist\n",
+    "\n",
+    "(x_train, y_train),(x_test, y_test) = mnist.load_data()\n",
+    "x_train, x_test = x_train / 255.0, x_test / 255.0\n",
+    "\n",
+    "def create_model():\n",
+    "    return tf.keras.models.Sequential([\n",
+    "        tf.keras.layers.Flatten(input_shape=(28, 28)),\n",
+    "        tf.keras.layers.Dense(512, activation='relu'),\n",
+    "        tf.keras.layers.Dropout(0.2),\n",
+    "        tf.keras.layers.Dense(10, activation='softmax')\n",
+    "      ])\n",
+    "\n",
+    "model = create_model()\n",
+    "model.compile(optimizer='adam',\n",
+    "              loss='sparse_categorical_crossentropy',\n",
+    "              metrics=['accuracy'])\n",
+    "\n",
+    "log_dir = \"logs/fit/\" + datetime.datetime.now().strftime(\"%Y%m%d-%H%M%S\")\n",
+    "tensorboard_callback = tf.keras.callbacks.TensorBoard(log_dir=log_dir, histogram_freq=1)\n",
+    "\n",
+    "model.fit(x=x_train, \n",
+    "          y=y_train, \n",
+    "          epochs=5, \n",
+    "          validation_data=(x_test, y_test), \n",
+    "          callbacks=[tensorboard_callback])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "tags": [
+     "header"
+    ]
+   },
+   "source": [
+    "## Step 1: Set your Environment Variables"
```

Comment:
This step probably diverges form the main path of the example.
Maybe consider saying sth about token and project and link to docs?

Reply:
I think in the first part (convert your logdir) this is actually an important, crucial step